### ISSUE TITLE
View full channel description in search results

### ISSUE THREAD

**Issue by opusforlife2** [TeamNewPipe/NewPipe#6474]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

I wish to read the full description for a channel in search results.

Possible methods:
- Long-pressing on the channel item expands it to show the full description.
- Long-pressing on the channel item opens a dialog which lets you view the description.

#### Is your feature request related to a problem? Please describe it

Firstly, Newpipe (to my knowledge) currently doesn't show you the channel description anywhere except for search results, so the only way to read it if you're interested is to search for that channel. Secondly, the description is limited to 2 lines only, and the rest is truncated with an ellipsis.

One way to read more of the text is to rotate to landscape. But even that may not be enough for some descriptions.

#### How will you/everyone benefit from this feature?

Ability to be informed what the channel is all about without having to read multiple video titles or descriptions, or worse, watch several videos, just to find out.

**Comment by skyGtm** [comment]
Looks similar to #5746

>Long-pressing on the channel item opens a dialog which lets you view the description.

This implementation looks good too

**Comment by opusforlife2** [comment]
Yeah, channel tabs is a major project. It would need a lot of time and effort. This is a short term solution to the problem.
